ci/coverage_publish.sh

```diff
@@ -34,7 +34,7 @@ if [ "${TRAVIS_SECURE_ENV_VARS}" == "true" ]; then
   echo "aws_secret_access_key=${COVERAGE_AWS_SECRET_ACCESS_KEY}" >> ~/.aws/config
   echo "region=us-east-1" >> ~/.aws/config
 
-  aws s3 cp "${COVERAGE_DIR}" "s3://${S3_LOCATION}" --recursive --profile coverage --acl public-read --quiet
+  aws s3 cp "${COVERAGE_DIR}" "s3://${S3_LOCATION}" --recursive --profile coverage --acl public-read --quiet --sse
```

Comment:
Based on my conversation with @mkercher this should still be visible in the browser. S3 should decrypt the object and make it visible per the access policy. Given that this is meant to be public, I guess we gain nothing by doing this. 

@mkercher, thoughts?